apispec/openapi.json

```diff
@@ -5236,23 +5236,23 @@
       "transport.cName": {
         "type": "string",
         "title": "cName",
-        "description": "The cName HDT generates for a transport, customer could use the cName to redirect the traffic to HDT platform.",
+        "description": "The cName HDT generates for a transport. Customer could use the cName to redirect the traffic to the HDT platform.",
```

Comment:
"cName" should be "CNAME".
That's what the DNS record is called.  It's never referenced as "cName" as far as I know.